### PR TITLE
Updated CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by gargarchit**
I found some Improper word choice as well as Improper formatting for that I have made changes and corrected that so, now It will be easier for everyone to read.

## To-do list

- [x] If my change impacts other components, I have tested to make sure they don't break.
- [x] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [x] This pull request is ready to merge.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Our Pledge
 
 In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
+contributors and maintainers pledge to make participation in our project and
```
Comment (GarthDB) [inline CODE_OF_CONDUCT.md]: good catch.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -48,17 +48,17 @@ threatening, offensive, or harmful.
 This Code of Conduct applies both within project spaces and in public spaces
 when an individual is representing the project or its community. Examples of
 representing a project or community include using an official project e-mail
-address, posting via an official social media account, or acting as an appointed
+address, posting via an official social media account or acting as an appointed
```
Comment (GarthDB) [inline CODE_OF_CONDUCT.md]: We use the serial comma (aka the Oxford or Harvard comma) so I think this comma should be put back in.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -48,17 +48,17 @@ threatening, offensive, or harmful.
...
 representative at an online or offline event. Representation of a project may be
 further defined and clarified by project maintainers.
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
+Instances of abusive, harassing or otherwise, unacceptable behavior may be
```
Comment (GarthDB) [inline CODE_OF_CONDUCT.md]: Same with this comma.

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -48,17 +48,17 @@ threatening, offensive, or harmful.
...
 reported by contacting the project team at Grp-opensourceoffice@adobe.com. All
 complaints will be reviewed and investigated and will result in a response that
 is deemed necessary and appropriate to the circumstances. The project team is
-obligated to maintain confidentiality with regard to the reporter of an incident.
+obligated to maintain confidentiality concerning the reporter of an incident.
```
Comment (GarthDB) [inline CODE_OF_CONDUCT.md]: This looks good.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (gargarchit) [reply CODE_OF_CONDUCT.md]: Thank You

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (gargarchit) [reply CODE_OF_CONDUCT.md]: Added back

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (gargarchit) [reply CODE_OF_CONDUCT.md]: Added

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (gargarchit) [reply CODE_OF_CONDUCT.md]: Thank You @GarthDB

**Comment by gargarchit** [comment]
Thank You @GarthDB for Reviewing and Merging my pull request

**Comment by GarthDB** [comment]
Thanks, @gargarchit !

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -54,11 +54,11 @@ further defined and clarified by project maintainers.
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
+Instances of abusive, harassing, or otherwise, unacceptable behavior may be
```
Comment (lazd) [inline CODE_OF_CONDUCT.md]: This is incorrect grammar. That comma is not necessary.

**Comment by lazd** [comment]
@GarthDB I don't think this PR should have been merged; it edits our Code of Conduct, which was sourced from the Open Source team and approved by legal.

**Comment by filmaj** [comment]
@lazd @GarthDB I think the edits are OK, though yes, not a huge amount of value in them.

@gargarchit if you want to edit the underlying documents, as @lazd pointed out, feel free to send a PR to [our starter repo](https://github.com/adobe/starter-repo) (where the code of conduct is revisioned) and I can work with you on the edits there.

**Pull request merged**